MIDIchlorians/AudioClipPlayer.swift

```diff
@@ -15,10 +15,11 @@ struct AudioClipPlayer {
 
     //creates ID and returns it to be mapped
     static func initAudioClip(audioDir: String, ext: String = "wav") -> UInt32? {
-
-        guard let soundURL = Bundle.main.url(forResource: audioDir, withExtension: ext) else {
+        guard let docsURL = FileManager.default.urls(for: .documentDirectory,
```

Comment:
can you try a fresh install of the app and make sure that the samples can still be played?
this might fail if the code I wrote in appdelegate copied to the wrong directory.